package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "start": "nodemon src/app.js",
-    "test": "yarn validate && jest --passWithNoTests",
+    "test": "set IS_JEST=1 && yarn validate && jest --passWithNoTests",
```

Comment:
I have to keep it this way in my computer, because git bash is not recognizing IS_JEST alone and due to that I am not able to push
I will change it directly from GitHub

Reply:
@Shrreya it is still connecting to the normal database not the test one 😕

Reply:
I searched for a way to set it in ubuntu and found one and it worked here as well 😁

Reply:
@AmmarBaki2 I think in the first change you left a `&&` in the command.
It should have been `IS_JEST=1 yarn validate && jest --passWithNoTests`

Anyway the `export` approach also works. Thank you for looking into this. I really appreciate your solution-oriented approach.

Reply:
Thank you for the motivation 😁